controllers/getReviewId.controller.js

```diff
@@ -0,0 +1,10 @@
+const { selectReviewId } = require(`../models/selectReviewId.model`);
+
+exports.getReviewId = (req, res, next) => {
+  selectReviewId(req.params.review_id)
+    .then((review) => {
+      const reviewToSend = review;
+      res.status(200).send({ reviewToSend });
```

Comment:
Keep the key of what you send back to the user simple and descriptive of what you are sending back, in the case of this function it is a single review, therefore I would encourage in your .send() you just return {review: review} instead of {reviewToSend}.